app/views/service_manual/index.html.erb

```diff
@@ -1,47 +1,55 @@
-<% content_for :title, "Service Manual" %>
-
-<header class="app-hero">
-  <div class="app-hero__content govuk-width-container">
-    <div class="govuk-main-wrapper">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-          <%= render "govuk_publishing_components/components/heading", {
-            text: "Service Manual",
-            font_size: "xl",
-            heading_level: 1,
-            inverse: true,
-            margin_bottom: 6,
+<%
+  content_for :title, "Service Manual"
+  content_for :main_classes, "govuk-!-padding-top-0"
+%>
```

Comment:
The meta description needs to be added separately too.

It populates this field in the page source:

```html
<meta name="description" content="Helping government teams create and run great digital services that meet the Service Standard.">
```

```ruby
<% content_for :extra_headers do %>
  <meta name="description" content=content="<%= strip_tags(@content_item.description) %>">
<% end %>
```

Reply:
The existing Service Manual page doesn't have open graph meta tags, so it's probably to leave them off here too.